apps/site/lib/site_web/templates/layout/_footer.html.eex

```diff
@@ -78,30 +78,35 @@
         </ul>
       </li>
       <li class="m-footer__group">
-        <div class="m-footer__group-title">Follow Us Online</div>
-        <ul class="c-social">
-            <% social_links = [
-                {"facebook", "https://www.facebook.com/TheMBTA/", []},
-                {"instagram", "https://www.instagram.com/thembta/", []},
-                {"linkedin", "https://linkedin.com/company/mbta", []},
-                {"youtube", "https://www.youtube.com/user/mbtagm/", []},
-                {"twitter", "https://twitter.com/MBTA", []},
-                {"tiktok", "https://www.tiktok.com/@thembta", [font_family: "fab"]}
-             ] %>
-          <%= for {name, href, attributes} <- social_links do %>
-            <li class="c-social__item m-footer__item">
-              <%= link [
-                fa(name, attributes),
-                content_tag(:span, "MBTA " <> String.capitalize(name), class: "sr-only")
-              ], to: href, class: "c-social__link c-social--" <> name, target: "_blank" %>
-            </li>
-          <% end %>
-        </ul>
         <%= unless assigns[:exclude_google_translate] do %>
           <%= render "_google_translate_plugin.html", conn: @conn %>
         <% end %>
       </li>
     </ul>
+
+    <div class="lower-footer">
+        <div class="m-menu__language"></div>
+      <div class="social-icons-container">
+          <a href="https://www.facebook.com/TheMBTA/" target="_blank">
+            <i aria-label="facebook" class="fa-brands fa-facebook fa-lg social-icon"></i>
+          </a>
+          <a href="https://www.instagram.com/thembta/" target="_blank">
+            <i aria-label="instagram" class="fa-brands fa-instagram fa-lg social-icon"></i>
+          </a>
+          <a href="https://linkedin.com/company/mbta" target="_blank">
+            <i aria-label="linkedin" class="fa-brands fa-linkedin fa-lg social-icon"></i>
+          </a>
+          <a href="https://www.youtube.com/user/mbtagm/" target="_blank">
+            <i aria-label="youtube" class="fa-brands fa-youtube fa-lg social-icon"></i>
+          </a>
+          <a href="https://twitter.com/MBTA" target="_blank">
+            <i aria-label="twitter" class="fa-brands fa-twitter fa-lg social-icon"></i>
+          </a>
+          <a href="https://www.tiktok.com/@thembta" target="_blank">
+            <i aria-label="tiktok" class="fa-brands fa-tiktok fa-lg social-icon"></i>
```

Comment:
Thanks for the `aria-label`! One thing - the old labels were formatted `MBTA [social platform name]` and I'd like these to do the same -- "MBTA tiktok" and "MBTA instagram" and so on will make much more sense to users than the "tiktok" label alone (does it go to www.tiktok.com?). Hope this makes sense.